### ISSUE TITLE
Verify the application is accessible

### ISSUE THREAD

**Issue by hortongn** [uclibs/aaec#8]
The form should be tested for accessibility.

**Comment by bsp3ars** [comment]
@hortongn Contact Lisa to test the site

**Comment by hortongn** [comment]
@bsp3ars Apparently this is the Chrome extension we used to test Scholar for accessibility issues.

After we made corrections for that, we submitted the app to IT@UC to be tested as well (but we probably won’t go that far for AAEC)


______________________________
From: "Bareswilt, Noelle (colemaj6)" <colemaj6@ucmail.uc.edu>
Date: Tuesday, September 5, 2017 at 2:43 PM
To: "Horton, Glen (hortongn)" <hortongn@ucmail.uc.edu>
Subject: RE: Quick question about accessibility scan

Absolutely. 
 
The plugin we use is called Attest and it is for Chrome.
https://chrome.google.com/webstore/detail/worldspace-attest-devtool/hhgcpikhbdemplnpnejejkfcpkhcajjm?utm_source=chrome-app-launcher-info-dialog
 
It may be more helpful for me to demo how to use it than to try and explain it through an email. 
 
You can use the plugin without connecting it to WorldSpace to scan for issues page by page.  However, if you connect it to WorldSpace, it allows you to upload issues to your project with labels like the QA team did in the first place (I’m not sure who exactly ran it the first time) and update issues as you fix them.  The plugin is also what we use to record scripts for more complicated sites. 
 
Noelle Bareswilt, Quality Assurance Analyst
UCIT Software Applications Development

**Comment by Janell-Huyck** [comment]
https://ucdts.atlassian.net/browse/LIBAAEC-26
